lib/cartopy/mpl/geoaxes.py

```diff
@@ -1537,6 +1537,32 @@ def scatter(self, *args, **kwargs):
         self.autoscale_view()
         return result
 
+    @_add_transform
+    def hexbin(self, x, y, *args, **kwargs):
+        """
+        Add the "transform" keyword to :func:`~matplotlib.pyplot.hexbin'.
+
+        The provided points are transformed into the axes projection and then
+        hexagons are constructed based on those transformed points.
```

Comment:
Minor wording suggestion.
```suggestion
        The points are first transformed into the projection of the axes and then
        the hexbin algorithm is computed using the data in the axes projection.
```